supreme_court_predictions/models/logistic.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import pandas as pd
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import train_test_split
+
+from supreme_court_predictions.util.files import get_full_pathway
+
+
+class LogisiticRegression:
+    def __init__(self):
+        self.local_path = get_full_pathway(
+            "/supreme_court_predictions/data/clean_convokit/"
+        )
+
+        self.utterances_df = pd.read_pickle(self.local_path + "utterances_df.p")
```

Comment:
This part may not work anymore since the file names are different depending on the token aggregation:
"case_aggregations.p" for all the cases
"advocate_aggregations.p": only for the utterances of speakers who are advocates
"adversary_aggregations.p": only for the utterances of speakers who are adversaries
"judge_aggregations.p": only for the utterances of speakers who are judges

This will be updated with my PR!

Reply:
also the local_path will need to be "/supreme_court_predictions/data/processed/"

Reply:
you're right. Right now, it just provides sample logistic regression. Can definitely improve so that it works with aggregations.p